optuna/integration/allennlp.py

```diff
@@ -21,6 +21,32 @@
     TrackerCallback = object
 
 
+def _save_best_config(config_file: str, study: optuna.Study) -> Dict:
```

Comment:
This function actually doesn't save anything so how about renaming like this?
```suggestion
def _dump_best_config(config_file: str, study: optuna.Study) -> Dict:
```

Reply:
Thank you for the suggestion, it makes sense (reanamed).